File: SDKs/googleapis.com/my-business/v4/java/lib/src/main/java/openapisdk/models/shared/FetchVerificationOptionsResponse.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * FetchVerificationOptionsResponse
 * Response message for Verifications.FetchVerificationOptions.
**/
public class FetchVerificationOptionsResponse {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("options")
    public VerificationOption[] options;
    public FetchVerificationOptionsResponse withOptions(VerificationOption[] options) {
        this.options = options;
        return this;
    }
}

File: SDKs/googleapis.com/mybusinessverifications/v1/python/src/sdk/models/shared/fetchverificationoptionsresponse.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import verificationoption as shared_verificationoption


@dataclass_json
@dataclasses.dataclass
class FetchVerificationOptionsResponse:
    r"""FetchVerificationOptionsResponse
    Response message for Verifications.FetchVerificationOptions.
    """
    
    options: Optional[list[shared_verificationoption.VerificationOption]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('options') }})
    
